airbyte-integrations/bases/base-normalization/normalization/transform_catalog/utils.py

```diff
@@ -118,7 +118,8 @@ def is_simple_property(definition: dict) -> bool:
 def is_combining_node(properties: dict) -> Set[str]:
```

Comment:
Some really unfair comments here so sorry in advance (non are blocking):

- `is_combining_node` reads like it should be a boolean, but it returns a set?
- As a new observer, I understand what its doing (checks a condition and returns an empty or populated set) but i have no idea why or what a "combining node" is, some docs on the method could be useful here
- is there a test for this?
- I don't feel too strongly about this but the return values _could_ be static but this is a super nitpick